clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -1089,9 +1089,10 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
     deleteUnassignedPartitionsOnStartup =
         serverProperties.getBoolean(SERVER_DELETE_UNASSIGNED_PARTITIONS_ON_STARTUP, false);
     aclInMemoryCacheTTLMs = serverProperties.getInt(ACL_IN_MEMORY_CACHE_TTL_MS, -1); // acl caching is disabled by
+                                                                                     // default
     aaWCIngestionStorageLookupThreadPoolSize =
         serverProperties.getInt(SERVER_AA_WC_INGESTION_STORAGE_LOOKUP_THREAD_POOL_SIZE, 4);
-    this.isParticipantMessageStoreEnabled = serverProperties.getBoolean(PARTICIPANT_MESSAGE_STORE_ENABLED, false);
+    this.isParticipantMessageStoreEnabled = serverProperties.getBoolean(PARTICIPANT_MESSAGE_STORE_ENABLED, true);
```

Comment:
curious, do we ever use false in prod ?

Reply:
No, we always use `true` in prod, since several years ago. I plan to fully delete this code in a future commit, but I want to deprecate it first, so we run the test suite in this way for some time. This way if we need to debug any test flakiness we can still go back to HMC and see if the flakiness is related to that.